lib/pages/labels/label_widget.dart

```diff
@@ -71,29 +71,45 @@ class LabelRow extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final homeBloc = context.bloc<HomeBloc>();
-    return ListTile(
-      key: ValueKey("tile_${label.name}_${label.id}"),
-      onTap: () {
-        homeBloc.applyFilter("@ ${label.name}", Filter.byLabel(label.name));
-        context.safePop();
+    return Dismissible(
```

Comment:
Need test for this widget

Reply:
ok, test done

Reply:
I cannot find the tests.

Reply:
the test is located in `WhatTodo\test\label_dismissible_widget_test.dart`